src/pages/[platform]/build-a-backend/storage/set-up-storage/index.mdx

```diff
@@ -100,6 +100,125 @@ export const storage = defineStorage({
 });
 ```
 
+### Configure additional storage buckets
+Amplify Storage allows you to optionally configure your backend to automatically create and manage multiple storage resources. 
+
+Start by creating additional storage resources in your backend using the `defineStorage` function and providing a `name` for your storage bucket. The `name` attribute is required and we recommend assigning a developer friendly value for each storage bucket to allow you to easily reference them from your application code later. You can define any custom access controls here as well.
+
+Note that if you define additional storage buckets one of them must be marked with `isDefault: true`.
+
+```ts title="amplify/storage/resource.ts"
+export const primaryStorage = defineStorage({
+  name: 'primaryStorage',
+  isDefault: true, // identify your default storage bucket (required)
+});
+
+export const secondaryStorage = defineStorage({
+  name: 'secondaryStorage',
+  access: (allow) => ({
+    'private/{entity_id}/*': [allow.entity('identity').to(['read', 'write', 'delete'])]
+  })
+})
+```
+
+Add your additional storage resources to your backend definition.
+
+```ts title="amplify/backend.ts"
+import { defineBackend } from '@aws-amplify/backend';
+import { auth } from './auth/resource';
+import { primaryStorage, secondaryStorage } from './storage/resource';
+
+defineBackend({
+  auth,
+  primaryStorage,
+  secondaryStorage
+});
+```
+
+Additional storage buckets can easily be referenced from your application code in Amplify Storage by passing the bucket option. The `bucket` option can either be a string representing the target bucket's assigned name in Amplify Backend or an object specifying the bucket name and region from the console as strings.
```

Comment:
```suggestion
Additional storage buckets can easily be referenced from your application code in Amplify Storage by passing the `bucket` option. You can reference the bucket either by providing the target bucket's assigned name in Amplify Backend or by specifying the actual bucket name and region from the console.
```

I suggest this, since not all libraries will have same type info and can refer specific type info in API section